src/openrct2/world/Duck.cpp

```diff
@@ -88,7 +88,7 @@ rct_duck * rct_sprite::AsDuck()
     rct_duck * result = nullptr;
     if (IsDuck())
     {
-        return result;
+        return (rct_duck *)this;
```

Comment:
This will require network version change

Reply:
No, it doesn't. This function is not used.

Reply:
ok